darwin/cli_functions.py

```diff
@@ -189,30 +193,30 @@ def path(dataset_slug: str) -> Path:
     )
 
 
-def url(dataset_slug: str) -> Path:
-    """Returns the url of the specified dataset"""
-    client = _load_client(offline=True)
+def url(dataset_slug: str) -> None:
+    """Prints the url of the specified dataset"""
+    client: Client = _load_client(offline=True)
     try:
-        remote_dataset = client.get_remote_dataset(dataset_identifier=dataset_slug)
+        remote_dataset: RemoteDataset = client.get_remote_dataset(dataset_identifier=dataset_slug)
         print(remote_dataset.remote_path)
     except NotFound as e:
         _error(f"Dataset '{e.name}' does not exist.")
 
 
-def dataset_report(dataset_slug: str, granularity) -> Path:
-    """Returns the url of the specified dataset"""
-    client = _load_client(offline=True)
+def dataset_report(dataset_slug: str, granularity) -> None:
+    """Prints the url of the specified dataset"""
+    client: Client = _load_client(offline=True)
     try:
-        remote_dataset = client.get_remote_dataset(dataset_identifier=dataset_slug)
-        report = remote_dataset.get_report(granularity)
+        remote_dataset: RemoteDataset = client.get_remote_dataset(dataset_identifier=dataset_slug)
+        report: str = remote_dataset.get_report(granularity)
         print(report)
     except NotFound:
         _error(f"Dataset '{dataset_slug}' does not exist.")
 
 
 def export_dataset(
-    dataset_slug: str, include_url_token: bool, annotation_class_ids: Optional[List] = None, name: Optional[str] = None
-):
+    dataset_slug: str, include_url_token: bool, name: str, annotation_class_ids: Optional[List] = None
```

Comment:
The way I see it is: `List`, `Tuple`, `Dict`, etc should always specify its items' types.
In this case, I believe this would be
```suggestion
    dataset_slug: str, include_url_token: bool, name: str, annotation_class_ids: Optional[List[int]] = None
```

Reply:
Any reasons why we swapped `name` and `annotation_class_ids`?